test/data/provision_for_mass.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "measurement_mode": "mass",
```

Comment:
just curious why `"measurement_mode"` is sorted to the top here when set to `"mass"`, but not for `"volume"`, when `json.dumps(sort_key=True)` sorts by keys.

Reply:
In json file can have the keys in any order. But at load time(with the option of `sort_keys=True`), it will sort the keys. 

The main reason for specifying the option of `sort_keys=True` is that, when developer defines json(with keys in any order) and there is no library to support comparison of logical structure and data of expected v/s actual json, I am relying on string comparison(for which keys need to be sorted)

Reply:
Got it, makes total sense. Since we can have these keys in any order, can we move `measurement_mode` to follow the line `"op": "provision",` just to harmonize with other provision json files?

Reply:
Ensured same order of keys across all files